setup.py

```diff
@@ -156,7 +156,7 @@ def run(self):
         prefix = os.path.abspath(self.prefix)
         self.set_data_files(prefix)
         self.do_egg_install()
-        self.ooniresources()
+        #self.ooniresources()
```

Comment:
As far as I could tell, the ooniresources call would download the data files every time the install action was performed, which uses a lot of bandwidth.  We disabled this part of the setup script and downloaded the resources as part of the ansible server setup.  Making the ooniresources call a separate action in setup.py (or using a command flag) would work well.  I think the vagrant setup uses the ooniresources script directly, rather than relying on the install action.